tests/integration/api_v3/http/posStakers.test.js

```diff
@@ -49,6 +49,20 @@ describe('Stakers API', () => {
 			expect(response.data.stakers.length).toBeLessThanOrEqual(10);
 		});
 
+		it('Returns list of stakers when requested for known validator address and search param (exact staker name)', async () => {
+			const response = await api.get(`${endpoint}?address=${refValidator.address}&search=${refValidator.name}`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data.stakers.length).toBe(1);
+		});
+
+		it('Returns list of stakers when requested for known validator address and search param (partial staker name)', async () => {
+			const searchParam = refValidator.name ? refValidator.name[0] : '';
+			const response = await api.get(`${endpoint}?address=${refValidator.address}&search=${searchParam}`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data.stakers.length).toBeGreaterThanOrEqual(1);
+			expect(response.data.stakers.length).toBeLessThanOrEqual(10);
```

Comment:
If (meta.count <= meta.total), please assert that one of the returned `stakerAddress` is an exact match for `refValidator.address`.

Reply:
Applied.